File: QuickApp/ClientApp/src/app/models/Intervention.ts
export class Intervention {
    public  id:number

    public  date:string;

    public  typeOfEventId:number

    public  nameOfEvent:string

    public  briefDescription: string

    public  mainStakeholdersParticipated:string

    public  apCategoryId:number

    public  numberOfParticipants:number

    public selectedPersons:Array<any> = new Array<any>();

    public  participants: Array<EventwiseParticipants> = new Array<EventwiseParticipants>();
}


export class EventwiseParticipants{

    public  id:number

    public  eventId:number

    public  participantId:number



}

export class SelectedPersonsData{
    public person_ID:number;
}

File: DAL/Models/Genaral/Intervention.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models.Genaral
{
     public class Intervention
    {
        public int Id { get; set; }

        public DateTime Date { get; set; }


        public int TypeOfEventId { get; set; }

        public string NameOfEvent { get; set; }

        public string BriefDescription { get; set; }

        public string MainStakeholdersParticipated { get; set; }

        public int ApCategoryId { get; set; }

        public int NumberOfParticipants { get; set; }

        public List<EventwiseParticipants> Participants { get; set; }
    }
}
